examples/dataservice-write/example.cpp

```diff
@@ -40,6 +40,7 @@ const std::string kLayer("");            // layer name inside catalog to use
 const std::string kData("hello world");  // data to write
 
 constexpr auto kLogTag = "write-example";
+constexpr size_t kSizePublish = 5u;
```

Comment:
```suggestion
constexpr size_t kPublishRequestsSize = 5u;
```
Or, I don't know, maybe `kNumRequestsToPublish` or `kRepeatPublishRequestCount`, cause current name is a bit unclear.